tests/acceptance/features/apiWebDavOperations/getFileByFileId.feature

```diff
@@ -0,0 +1,109 @@
+@api
```

Comment:
@PrajwolAmatya maybe the suite name can be `apiSpacesDavOperation` ? I see we are using `  Given using spaces DAV path` and we also already have `coreapiwebdavoperation` if we are going to add tests `new` and `old` dav path as well then the current name is okay.

Reply:
> @PrajwolAmatya maybe the suite name can be `apiSpacesDavOperation` ? I see we are using ` Given using spaces DAV path` and we also already have `coreapiwebdavoperation` if we are going to add tests for `new` and `old` dav path as well then the current name is okay.

These tests works only for `spaces DAV path`. I will rename the suite to `apiSpacesDavOperation`.